src/lib/chips/chip.spec.ts

```diff
@@ -184,6 +193,15 @@ describe('Chips', () => {
           expect(chipInstance.selected).toBeFalsy();
           expect(testComponent.chipSelect).not.toHaveBeenCalled();
         });
+
+        it('should have empty aria-selected', () => {
+          expect(chipNativeElement.getAttribute('aria-selected')).toBe('');
```

Comment:
Should this be `expect(chipNativeElement.hasAttribute('aria-selected')).toBe(false);`?